tokio/src/io/util/lines.rs

```diff
@@ -59,6 +59,21 @@ where
 
         poll_fn(|cx| Pin::new(&mut *self).poll_next_line(cx)).await
     }
+
+    /// Obtain a mutable reference to a byte of the underlying reader
+    pub fn get_mut(&mut self) -> &mut R {
+        &mut self.reader
+    }
+
+    /// Obtain a reference to a byte of the underlying reader
```

Comment:
Similarly, this borrows the reader, not "a byte of the reader":
```suggestion
    /// Obtain a reference to the underlying reader
```

Reply:
Resolved with https://github.com/tokio-rs/tokio/pull/2450/commits/7dc85efbf3f9d402682ab007f589d87316938bec